src/ext/php5/dispatch.c

```diff
@@ -0,0 +1,158 @@
+#include "dispatch.h"
+
+#include <Zend/zend_exceptions.h>
+#include <php.h>
+
+#include <ext/spl/spl_exceptions.h>
+
+#include "ddtrace.h"
+#include "debug.h"
+
+ZEND_EXTERN_MODULE_GLOBALS(ddtrace)
+
+// todo: is this used anywhere?
```

Comment:
Did we figure this one out? Perhaps this is for compatibility with PHP 5.4?

Reply:
It looks like a compatibility macro for PHP _5.3_.